.github/workflows/nightly-emscripten.yml

```diff
@@ -20,6 +20,14 @@ jobs:
       nightly-already-exists: ${{ env.NIGHTLY_ALREADY_EXISTS }}
 
     steps:
+      - name: Remove unused software to free up space
+        run: |
+          # These are quick to delete and large enough to make a difference
+          rm -rf /usr/share/swift/                        # 1.3 Gb in 80 subdirs
+          rm -rf /usr/local/lib/android/sdk/build-tools   # 2.1 Gb in 450 subdirs
+          rm -rf /usr/share/dotnet/shared                 # 5.3 Gb in 350 subdirs
+          rm -rf /usr/local/lib/android/sdk/ndk           # 7.6 Gb in 1500 subdirs
```

Comment:
I'd prefer to put it in a reusable script instead of repeating it in all actions but unfortunately then I could only access it after I already have a checkout of the repository.

I could retrieve it with `curl` via github API but I'm not sure if complicating it that way is worth it.

Reply:
@cameel you can put that into an action and just make use of it here, that gives you less copy'n'pasting and accidentally forgetting something and a central way to improve the script without touching it in every used repository.

Reply:
You mean create a reusable action? Not a bad idea. I'll try that.

Reply:
i just tried that and unfortunately it won't work without putting the action in a separate repo. The problem is that actions from the same repo do not get checked out automatically and to check it out you need to check out the whole repo which is too big in our case.